strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/dependency/snippet/NpmDependencyFormatterTest.java

```diff
@@ -11,19 +11,22 @@
 
 /**
  * @author carlspring
+ * @author Pablo Tirado
  */
 @Execution(CONCURRENT)
 public class NpmDependencyFormatterTest
 {
+    private DependencySynonymFormatter formatter;
 
+    @BeforeEach
+    public void initialize() {
+        formatter = new NpmDependencyFormatter();
+        assertNotNull(formatter, "Failed to look up dependency synonym formatter!");
```

Comment:
actually this assertion looks strange, it can't be `null` anyway from what I see :)

I suggest just to drop this `initialize` method and and just do it like this:
```
    private DependencySynonymFormatter formatter = new NpmDependencyFormatter();
```